openc3-cosmos-init/plugins/packages/openc3-tool-common/src/tools/admin/tabs/PluginsTab.vue

```diff
@@ -29,7 +29,7 @@
           show-size
           accept=".gem"
           class="mx-2"
-          label="Click to select plugin .gem file to install"
+          label="Click to install new plugin .gem (NOT upgrade)"
```

Comment:
I wanted this to be clearer that this is for installing new plugins. I think we're still getting people clicking here for upgrade which leads to the "already exists" error.

Reply:
The real fix for this is a new "Are you sure" prompt that lets the user know that the plugin they are trying to install has already been installed at least once.   This info could be returned in the results from install_phase1